cmd/harmony/main.go

```diff
@@ -6,6 +6,8 @@ import (
 	"io/ioutil"
 	"math/big"
 	"math/rand"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
does this import do side effects? will it generate slower code?

Reply:
`_ "net/http/pprof"` simply registers the /debug/pprof/ endpoints via pprof's init() function.

"net/http" already gets pulled in by internal/memprofiling/lib.go, internal/profiler/profiler.go and staking/webhooks/yaml.go - so "net/http" will be included in the binary anyways.

If the imports in those files haven't slowed down the binary so far, then I assume the imports for this PR won't slow it down either.